gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/BlockTripsWithInconsistentRouteTypesValidatorTest.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.BlockTripsWithInconsistentRouteTypesNotice;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsRoute;
+import org.mobilitydata.gtfsvalidator.table.GtfsRouteTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsRouteType;
+import org.mobilitydata.gtfsvalidator.table.GtfsTrip;
+import org.mobilitydata.gtfsvalidator.table.GtfsTripTableContainer;
+
+@RunWith(JUnit4.class)
+public class BlockTripsWithInconsistentRouteTypesValidatorTest {
+  private BlockTripsWithInconsistentRouteTypesValidator validator =
+      new BlockTripsWithInconsistentRouteTypesValidator();
+  private final GtfsTrip trip11 =
```

Comment:
Nit:
This test would be more readable if the trip names were more informative.

Perhaps something like _trip_route_r1_block_apple_

Reply:
Thank you for the advice! It really makes sense. I already changed trips' names.